src/site/content/en/blog/access-devices/index.md

```diff
@@ -0,0 +1,161 @@
+There are many APIs to choose from. By answering a few questions on this device, you
+should find out the API to use and have some documentation to get started
+accessing this device.
+
+## 1. Do you want to access video stream from this device? {: #video }
+
+If so, use [`MediaDevices.getUserMedia()`] to get a live video stream from this
+device and learn about [capturing video]. You can also [control the camera's
+pan, tilt, and zoom], and other camera settings such as [brightness and
+contrast], and even [take still images].
+
+## 2. Do you want to access audio streams from this device? {: #audio }
+
+If so, use [`MediaDevices.getUserMedia()`] to get a live audio stream from this
+device and learn about [capturing audio]. You can also use [Web Audio] to add
+effects to audio, create audio visualizations, or apply spatial effects (such as
+panning). Check out [how to profile the performance of Web Audio apps] in Chrome
+as well.
+
+## 3. Do you want to receive input events from this device? {: #input }
+
+If so, try listening for [Keyboard] and [Pointer] events. If this device is a
+game controller, use the [Gamepad] API to know which buttons are being pressed
+and which axes moved.
+
+## 4. Is this device an NFC tag? {: #nfc }
+
+Then, try [Web NFC] to read and write to this device when it's in close
+proximity to the user's device (usually 5-10 cm, 2-4 inches).
+
+## 5. Is this a Bluetooth device? {: #bluetooth }
+
+Click the button below and see if you can spot a nearby Bluetooth device that
+may be of interest to you. Note that this device may need to be in a special
+mode to be visible.
+
+<div class="w-text--center">
+<button class="w-button w-button--secondary w-button--with-icon"
+data-icon="bluetooth" onclick="navigator?.bluetooth?.requestDevice({ acceptAllDevices:true })
+|| alert('This browser does not support Web Bluetooth. Try Chrome.')">
+Request bluetooth device
+</button>
+</div>
+
+If so, try [Web Bluetooth] to connect to this device over a Bluetooth Low Energy
+connection. It should be pretty easy to communicate with when it uses
+standardized Bluetooth GATT services (such as the battery service) as their
+behavior is [well-documented]. If not, at this point, you either have to find
+some hardware documentation for this device or reverse-engineer it. See the
+[Reverse-Engineering a Bluetooth Lightbulb] article from Uri Shaked for an
+example on how to do that.
+
+## 6. Does this device expose HID interfaces? {: #hid }
+
+Click the button below and see if you can spot a HID device that may be of
+interest to you. Note that security-sensitive HID devices (such as FIDO HID
+devices used for stronger authentication) are not accessible in Chrome.
+
+<div class="w-text--center">
+<button class="w-button w-button--secondary w-button--with-icon"
+data-icon="videogame_asset" onclick="navigator?.hid?.requestDevice({ filters: [] })
+|| alert('This browser does not support WebHID. Try Chrome on desktop.')">
+Request HID device
+</button>
```

Comment:
As cute as this is a device appearing in the chooser doesn't always mean that it is accessible. Tools like chrome://bluetooth-internals and https://webhid-collections.glitch.me/ can be really helpful for exploring what is actually possible a device.